HOW_TO_TEST.md

```diff
@@ -58,7 +58,7 @@ Molecule allows for testing a role and will live inside a role's directory in a
 
 To get a list of the scenarios:
 
-```ls roles/confluent.test/molecule```
+```ls molecule```
```

Comment:
```suggestion
```molecule list```
```
Gives more information. Could also keep it as is....

Reply:
im not sure i like the output of molecule list it shows all the containers, its a little overkill